charts/argo-cd/values.yaml

```diff
@@ -6,7 +6,8 @@ global:
     # -- If defined, a repository applied to all ArgoCD deployments
     repository: quay.io/akuity/argocd
     # -- If defined, a tag applied to all ArgoCD deployments
-    tag: v2.3.4-ak.0
+    # https://quay.io/repository/akuity/argocd?tab=tags
+    tag: v2.4.6-ak.0
```

Comment:
Instructions are updated in https://github.com/akuityio/akuity-enterprise once https://github.com/akuityio/akuity-enterprise/pull/23 is merged.

Reply:
Yeap, that was one of the questions I was planning to ask - when this tag is updated. Right now attempting to pull it from https://quay.io/repository/akuity/argocd?tab=tags would fail. Will check that other PR, thanks!